Added empty base snapshots to all test snapshots

agarwal-navin opened the pull request:
Added empty base snapshots to all the test snapshots. This should help us get rid of the `existing` property on Container. Currently, there is a workaround during container.load for these tests as they do not load from a snapshot.
With this, we will have a unified flow where every container loads from a base snapshot.

An additional advantage of this is that we can a metadata blob to these base snapshots to enable and run GC on all these snapshots which should give us additional test test coverage for GC.

andre4i reviewed (approved):
I don't know if these are correct, not familiar with the format. If you could point me to some documentation about how to get these or if you could specify how you've obtained them and why for these specific folders that would be great. I would love to understand the logistics of these files.

HOWEVER, I have a branch to remove the legacy `create-on-load` logic from the container ([here](https://github.com/microsoft/FluidFramework/compare/main...andre4i:remove-container-legacy-path?expand=1)) which I have successfully tested against this branch of the snapshot tests 🎉  so that is great and this is why I'm approving this change. Thank you!

agarwal-navin commented:
I added a README on how to add new snapshots to the snapshot tests in this PR - https://github.com/microsoft/FluidFramework/pull/6984. I followed the steps outlined in "Adding new snapshots" section.
Additionally, because these documents are supposed to start empty, I manually deleted the following:
- The .metadata blob from tree.json. These were added fairly recently so the old snapshots don't need it.
- All data stores' summaries from the tree.json.
- With that, only 3 blobs were left in the tree.json. I deleted the rest of the blob files (the ones named in format `number-id`.
- Lastly, I deleted the quorum values because for these old snapshots, the values come via ops.

As for why I added them to the `base_snapshot` folder - Replay tool can load documents from a base snapshot (downloaded via the fetch tool) and then replay ops on top of that. I recently made [this change](https://github.com/microsoft/FluidFramework/pull/6942) that allows specifying the base snapshot for snapshot tests under `base_snapshot` folder.
